webrender/src/lib.rs

```diff
@@ -202,4 +201,5 @@ pub use renderer::{RendererStats, SceneBuilderHooks, ThreadListener, ShaderPreca
 pub use renderer::MAX_VERTEX_TEXTURE_WIDTH;
 pub use shade::{Shaders, WrShaders};
 pub use webrender_api as api;
-pub use resource_cache::intersect_for_tile;
+pub use webrender_api::euclid;
```

Comment:
is this meant to be `pub`?

Reply:
yeah, I can't think of any reason to not make it public since users of webrender need to keep in sync with webrender's version of euclid. This way it's easier to just use `webrender::euclid` or `webrender_api::euclid` (depending on the situation) and not have to worry about having the right version the Cargo.toml.

In my opinion we should do this with all of the dependencies that are both in webrender and webrender_api, it's simpler to manage.

Reply:
I don't think it's possible to work with `webrender` but not link to `webrender_api`. Since the latter already exposes `euclid`, it would be more strict to just keep it the only crate that re-exports dependencies.

Reply:
I'm not sure what would be the advantage from being stricter here. For all consumers of WebRender except servo, the `webrender` crate is the only dependencies and the api is treated as module of it. It's rather common to expose public dependencies at the root of the crate. I wouldn't lose sleep over having to look for euclid and other dependencies in `webrender::api` rather than the root if you feel strongly about it but I'm curious about what the pros of that would be.

Reply:
ah, right, they access it through `webrender::api`. Ok, that's fine then